ReportUtilityBills/resources/simple_rates/README.md

```diff
@@ -18,3 +18,9 @@ Propane
 * https://www.eia.gov/dnav/pet/xls/PET_PRI_WFR_A_EPLLPA_PRS_DPGAL_W.xls
 * Convert second tab (Data 1) to CSV
 * PET_PRI_WFR_A_EPLLPA_PRS_DPGAL_W.csv
+
+Wood
+* https://www.eia.gov/state/seds/seds-data-fuel.php?sid=US#DataFiles
+* Download Prices (CSV) link
+* Filter the data where "MSN" is labeled as "WDRCD" (wood price in the residential sector)
+* SEDS_price_of_wood.csv
```

Comment:
Interesting. It looks like this data source has costs for all fuel types. Perhaps we can replace all the individual EPA files we were using before for different fuel types with this single download? It's possible that we just weren't aware of it (or it's a more recently available data source). Can you compare the prices in here to what we downloaded from the other sources and see if they are basically the same?

That would simplify this process of updating costs in the future if we could just download a single new file and drop it in.

Reply:
While electricity prices are the same, prices for other fuel types are quite different depending on the location. This raises doubts about whether using these wood prices is appropriate or not.